pkg/leakybucket/bucket.go

```diff
@@ -237,7 +239,15 @@ func LeakRoutine(leaky *Leaky) error {
 		case ofw := <-leaky.Out:
 			leaky.overflow(ofw)
 			return nil
-			/*we underflow or reach bucket deadline (timers)*/
+		/*suiciiiide*/
+		case <-leaky.Suicide:
+			close(leaky.Signal)
+			BucketsUnderflow.With(prometheus.Labels{"name": leaky.Name}).Inc()
```

Comment:
Should we considered this as underflow ? don't we want a `BucketCanceled` metric ?